addons/test_mail/tests/test_message_post.py

```diff
@@ -798,6 +800,39 @@ def test_message_post(self):
                 partner_ids=self.partner_portal.ids,
             )
 
+    @mute_logger('odoo.addons.mail.models.mail_mail', 'odoo.models.unlink')
+    @users('employee')
+    def test_message_post_using_controller(self):
+        """ Test mail_message_post of ThreadController (especially on archived record). """
+        # Use an archivable test record
+        test_record = self.env['mail.test.activity'].create({})
+        body_text = 'Log a note on record'
+        post_data = {
+            'body': body_text,
+            'message_type': 'comment',
+            'subtype_xmlid': 'mail.mt_comment',
```

Comment:
We could add ``partner_emails`` as it seems the controller support this argument and transforms it into partners

Reply:
Just wanted to test 'partner_emails", as it is specific to the controller. Could check created partners afterwards, improving code coverage :)